src/js/web-components/video-player/VideoPlayer.js

```diff
@@ -125,10 +125,18 @@ export default class extends HTMLElement {
         window.cast.framework.CastContextEventType.SESSION_STATE_CHANGED,
         async (e) => {
           if (e.sessionState === 'SESSION_STARTED' || e.sessionState === 'SESSION_RESUMED') {
+            const castableSources = this.internal.videoData['video-sources'].filter((source) => source.cast === true) || [];
```

Comment:
Is ` || []` required there?

Reply:
@beaufortfrancois No, well spotted, thank you. Updated in 1596a09.